### PR TITLE
feat: change handling labels & connect to state

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

handler changes
- to escape: 'Esc' keyboard click, click on outside mouse click
- to enter: 'Enter' keyboard click

style changes 
- max width 50%

and connect to each page's state

### CODE REVIEW

File: src/services/dashboards/modules/dashboard-label/DashboardLabels.vue [hunk 1]
```diff
@@ -51,22 +54,26 @@ const state = reactive({
...
+    state.inputText = '';
 };
+const handlePushLabel = () => {
+    state.labelList.push({ label: state.inputText });
+    state.inputText = '';
+};
+const handleDelete = (index: number) => {
+    state.labelList.splice(index, 1);
+};
+
```
Comment (WANZARGEN) [inline src/services/dashboards/modules/dashboard-label/DashboardLabels.vue:68]:
```suggestion
const handlePushLabel = () => {
    state.labelList.push({ label: state.inputText });
    state.inputText = '';
    emit('update:labelList', state.labelList);
};
const handleDelete = (index: number) => {
    state.labelList.splice(index, 1);
        emit('update:labelList', state.labelList);
};

```

**Review by WANZARGEN — COMMENTED** [review]
Nice clean code :)
I left some comments!

File: src/services/dashboards/modules/dashboard-label/DashboardLabels.vue [hunk 2]
```diff
@@ -1,14 +1,17 @@
 <template>
-    <div class="dashboard-labels"
-         @keydown="handleKeydown"
+    <div v-on-click-outside="handleEscape"
+         class="dashboard-labels"
+         @keydown.esc="handleEscape"
+         @keydown.enter="handlePushLabel"
```
Comment (yuda110) [inline src/services/dashboards/modules/dashboard-label/DashboardLabels.vue:5]: praise: very clear!

**Review by yuda110 — APPROVED** [review]
LGTM!
It would be nice if comments were reflected :)

**Review by WANZARGEN — APPROVED** [review]
LGTM 👍

File: src/services/dashboards/modules/dashboard-label/DashboardLabels.vue [hunk 3]
```diff
@@ -28,21 +31,21 @@
...
-    deleteable: boolean;
+    editable?: boolean;
 }
 const props = defineProps<Props>();
 
 const state = reactive({
-    // labelList: props.labelList,
-    // labelList: [{ label: 'AWS' }, { label: 'Taco' }],
-    labelList: [] as Array<{label: string}>,
+    labelList: props.labelList,
```
[the remark this answers is not in the document]
Comment (minseolee) [reply src/services/dashboards/modules/dashboard-label/DashboardLabels.vue]: I did not expect about it. I've applyed it. Thanks a lot !!

**Pull request merged**